src/controllers/message.controller.js

```diff
@@ -0,0 +1,42 @@
+const Message = require("../models/message.models");
+const SubChannel = require("../models/subChannel.models");
+const {response_500, response_400, response_200} = require("../utils/responseCodes.utils");
+
+
+exports.sendMessage = async (req, res) => {
+    try {
+        const {message, attachments, user} = req.body;
+        const subChannelId = req.params.id;
+
+        if (!message || !subChannelId || !user) {
+            return response_400(res, "Missing required fields");
+        }
+        console.log(subChannelId);
+        const subChannel = await SubChannel.findById(subChannelId);
+        if (!subChannel) {
+            return response_400(res, "Invalid SubChannel");
+        }
+
+        const newMessage = new Message({
+            message,
+            files: attachments.map((file) => file.file.toString()),
+            createdBy: user._id.toString(),
+            subChannel: subChannelId,
+        })
+
+        const savedMessage = await newMessage.save();
+
+        const updatedSubChannel = await SubChannel.findByIdAndUpdate(
```

Comment:
![image](https://github.com/opencodeiiita/Hitch-Backend/assets/105218991/4767f360-e468-4809-b0fc-1e197d0550ba)
Also update the latestMessage field